qoffer-infrastructure/src/main/groovy/life/qbic/portal/qoffer2/web/views/CreateOfferView.groovy

```diff
@@ -96,6 +96,7 @@ class CreateOfferView extends FormLayout{
         navigationView.showNextStep()
         this.addComponent(navigationView)
         this.addComponent(projectInformationView)
+        this.addComponent(createCustomerView)
```

Comment:
Why adding it here? AFIK here are only the views that are initially required. When commenting it out it still worked for me